lerobot-web-fe/src/components/CameraStream.tsx

```
@@ -1,39 +1,24 @@
-              </div>
-            ) : (
-              <div className={styles.noSignal}>
-                <ExclamationTriangleIcon className={styles.noSignalIcon} />
-                <span>No Signal!</span>
-              </div>
-            )}
-            <div className={styles.cameraLabel}>Camera {cameraId}</div>
-          </div>
-        ))}
+      {cameraIds && cameraIds.length > 0 && apiUrl ? (
+        cameraIds.map((cameraId) => <CameraBox id={cameraId} apiUrl={apiUrl} />)
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)